extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,9 +2,8 @@
 
 ## [UNRELEASED]
 
-
+- Add progress messages to LGTM download option. This makes the two-step process (selecting a project, then selecting a language) more clear.
 - Remove line about selecting a language from the dropdown when downloading database from LGTM. This makes the download progress visible when the popup is not expanded. [#894](https://github.com/github/vscode-codeql/issues/894)
```

Comment:
```suggestion
- Add progress messages to LGTM download option. This makes the two-step process (selecting a project, then selecting a language) more clear. [#960](https://github.com/github/vscode-codeql/pull/960)
- Remove line about selecting a language from the dropdown when downloading database from LGTM. This makes the download progress visible when the popup is not expanded. [#957](https://github.com/github/vscode-codeql/pull/957)
```

Very minor suggestion: Let's link to the pull request (rather than the issue), for consistency with the rest of the changelog entries 😃